internal/service/chat_service.go

```diff
@@ -387,3 +387,37 @@ func (s *ChatService) HasNextMessages(
 
 	return hasNext, nil
 }
+
+// 채팅 메시지를 저장합니다.
+func (s *ChatService) SaveChatMessage(
+	ctx context.Context, userID, roomID uuid.UUID, messageType, content string,
+) (*chat.Message, *pnd.AppError) {
```

Comment:
이거 리베이스하면 아마 오류날 거에요. 이렇게 바꿔주시면 작동할 것 같습니다.

```suggestion
) (*chat.Message, error) {
```